go/private/rules/test.bzl

```diff
@@ -262,14 +263,21 @@ _go_test_kwargs = {
         ),
         "rundir": attr.string(
             doc = """ A directory to cd to before the test is run.
-            This should be a path relative to the execution dir of the test.
+            This should be a path relative to the subdirectory of the runfiles
+            directory corresponding to the test's repository.
```

Comment:
Would it be more clear to say "This should be a path relative to the root directory of the workspace where the test is defined, which can be the main or an external workspace"?

Reply:
Good point, I reworded this to avoid mentioning runfiles explicitly (except in the Windows case where knowledge about this is necessary anyway)

Reply:
I am not using the term `workspace` anymore because it means `main repository` to some people and `any repository` to others and is (luckily) going away with Bzlmod.